server/public/model/integration_action.go

```diff
@@ -434,7 +434,7 @@ func encryptPostActionCookie(plain string, secret []byte) (string, error) {
 
 	sealed := aesgcm.Seal(nil, nonce, []byte(plain), nil)
 
-	combined := append(nonce, sealed...)
+	combined := append(nonce, sealed...) //nolint:makezero
```

Comment:
This linter exclusion is unfortunate, because the `nonce` is clearly written above. It's like the rule slices doesn't consider how `[]byte`s get used.